### DISCUSSION TITLE
Orchid ecosystem / third party packages

### DISCUSSION THREAD

**Discussion by bernhardh** [orchidsoftware/platform#1719] [General]
As I wrote in my last ticket, I am new to orchid, coming from laravel nova. One thing I really loved in nova was its third party ecosystem. If I was looking for any "special" field type, there was a good chance, that I found it on https://novapackages.com/

Is there anything like this for orchidsoftware? A place where third party developer (or the orchid team) can provide extra modules/packages for orchid?

**Comment by tabuna** [comment]
Hi @bernhardh, We were all newbies once, so don't worry about that. We discussed this several times in telegram chats ([there are links in the project description](https://github.com/orchidsoftware/platform/blob/master/readme.md)) and concluded that there are not so many packages so far. They can fit into a [separate repository](https://github.com/orchidsoftware/awesome). When there are a little more of them, I will create a section on the website.
